tests/unit/modules/dcnm/fixtures/dcnm_intf_pc_configs.json

```diff
@@ -428,7 +449,7 @@
       "switch": [
         "10.122.84.108"
```

Comment:
Private ip

Reply:
Ok. Will modify the code to use private IP addresses